source/_components/noaaweather.markdown

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: "NOAA/NWS Sensor"
+description: "How to integrate NOAA/National Weather Service observations within Home Assistant."
+date: 2019-04-15 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: 
```

Comment:
I have added already the variable for you, but could you add a logo of the integration?

Reply:
This integration will use the same logo(s) as the one in [#9372](https://github.com/home-assistant/home-assistant.io/pull/9372).  The same comment will apply that we need to request permission.  We are not using unmodified data products, both this platform/integration and the one in #9372 use minimally altered, but not unaltered data.

Reply:
I retract this, see comment in #9372